### ISSUE TITLE
Not reacting Dropbox button on arch

### ISSUE THREAD

**Issue by gustawdaniel** [keeweb/keeweb#1593]
I installed `keeweb` by

1. deb on ubuntu
2. yay on arch

I trying to connect with dropbox. On ubuntu it works:

![Zrzut ekranu z 2020-09-06 13-11-58](https://user-images.githubusercontent.com/16663028/92324513-bf869f00-f042-11ea-8004-279252a29a91.png)

But on arch console log shows different commands:

![Zrzut ekranu z 2020-09-06 13-14-25](https://user-images.githubusercontent.com/16663028/92324546-096f8500-f043-11ea-8dc2-c17e575d8497.png)

I clicking on dropbox icon after first opening of program.

**Comment by antelle** [comment]
Hi! What happens if you run keeweb from a zip file?

**Comment by gustawdaniel** [comment]
I opened from zip on arch:

![Zrzut ekranu z 2020-09-06 13-27-53](https://user-images.githubusercontent.com/16663028/92324745-c7474300-f044-11ea-8ee0-ffd357d2c4e7.png)

**Comment by antelle** [comment]
What happens after it, does it open a web browser or just hangs?

**Comment by gustawdaniel** [comment]
Option 2: just hangs.

I installed it from aur on second arch instance and I can't reproduce this problem.

**Comment by antelle** [comment]
Interesting, so it can't open the browser. Not sure why this happens but I can report it to Electron if I repeat it. What is the arch version?

**Comment by gustawdaniel** [comment]
I checked it again. There are logs from console of keeweb

```
index.html:1 2020-10-19T20:09:42.553Z [native-module-connector] Received an error start-usb {name: "Error", message: "The module '/usr/lib/keeweb/node_modules/@keeweb/k… instance, using `npm rebuild` or `npm install`).", stack: "Error: The module '/usr/lib/keeweb/node_modules/@k…/lib/keeweb/app.asar/native-module-host.js:24:21)"}
l.error @ index.html:1
result @ index.html:1
hostCallback @ index.html:1
(anonymous) @ index.html:1
emit @ events.js:310
emit @ internal/child_process.js:876
processTicksAndRejections @ internal/process/task_queues.js:85
index.html:1 Uncaught (in promise) {name: "Error", message: "The module '/usr/lib/keeweb/node_modules/@keeweb/k… instance, using `npm rebuild` or `npm install`).", stack: "Error: The module '/usr/lib/keeweb/node_modules/@k…/lib/keeweb/app.asar/native-module-host.js:24:21)"}
result @ index.html:1
hostCallback @ index.html:1
(anonymous) @ index.html:1
emit @ events.js:310
emit @ internal/child_process.js:876
processTicksAndRejections @ internal/process/task_queues.js:85
index.html:1 2020-10-19T20:09:43.004Z [native-module-connector] Host exited with code null and signal SIGSEGV
```

There is log from keeweb logs

```
[ERROR] 2020-10-19T20:09:06.197Z [native-module-connector] Received an error start-usb {"name":"Error","message":"The module '/usr/lib/keeweb/node_modules/@keeweb/keeweb-native-modules/usb-linux-x64.node'\nwas compiled against a different Node.js version using\nNODE_MODULE_VERSION 80. This version of Node.js requires\nNODE_MODULE_VERSION 82. Please try re-compiling or re-installing\nthe module (for instance, using `npm rebuild` or `npm install`).","stack":"Error: The module '/usr/lib/keeweb/node_modules/@keeweb/keeweb-native-modules/usb-linux-x64.node'\nwas compiled against a different Node.js version using\nNODE_MODULE_VERSION 80. This version of Node.js requires\nNODE_MODULE_VERSION 82. Please try re-compiling or re-installing\nthe module (for instance, using `npm rebuild` or `npm install`).\n    at process.func [as dlopen] (electron/js2c/asar.js:140:31)\n    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1196:18)\n    at Object.func [as .node] (electron/js2c/asar.js:140:31)\n    at Module.load (internal/modules/cjs/loader.js:981:32)\n    at Module._load (internal/modules/cjs/loader.js:881:14)\n    at Function.Module._load (electron/js2c/asar.js:769:28)\n    at Module.require (internal/modules/cjs/loader.js:1023:19)\n    at require (internal/modules/cjs/helpers.js:77:18)\n    at reqNative (/usr/lib/keeweb/app.asar/native-module-host.js:172:19)\n    at start-usb (/usr/lib/keeweb/app.asar/native-module-host.js:24:21)"}
[ERROR] 2020-10-19T20:09:06.678Z [native-module-connector] Host exited with code null and signal SIGSEGV
```

When I clicking Google Icon I can login and synchronize.

When I clicking Dropbox Icon I am redirected to link:

```
https://www.dropbox.com/oauth2/authorize?client_id=xxx&scope=&state=xxx8&redirect_uri=http%3A%2F%2Flocalhost%3A48149%2Foauth-result%2Fdropbox.html&response_type=code
```

and page have error with content:

```
Error (400)
It seems the app you were using submitted a bad request. If you would like to report this error to the app's developer, include the information below.


More details for developers
No scope requested can be granted for this app.
```

I can be connected with a fact I had node js installed by arch, but I removed it and now have installed by nvm.

I will check it now.

**Comment by antelle** [comment]
Wow, which version of KeeWeb is that?

**Comment by gustawdaniel** [comment]
I removed keeweb by `yay` and from `snap` too. Then I recompiled by 

```
yay -S keeweb
```

and selected all default options.

---

Finally issue is solved. Reason of problem was steps:

1. Installing node js on arch by yay.
2. Instaling keeweb by arch.
3. Removing node js from aur and installing by `nvm`.

---

I fixed by:

1. Removing by `yay`
2. Removing by `snap`
3. Installing `keeweb`, no `keeweb-desktop-bin` by `yay`, so after compilation it works great.

Thank for your support @antelle :+1:
